src/backend/distributed/citus--5.1-3--5.1-4.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR REPLACE FUNCTION worker_apply_shard_ddl_command(bigint, text, text)
```

Comment:
ALTER FUNCTION does not work for changing parameter count, instead I dropped and recreated the function